### DISCUSSION TITLE
Connecting SecurityOnion to my home network -- wired vs wireless (WiFi)

### DISCUSSION THREAD

**Discussion by red-r3b3l** [Security-Onion-Solutions/securityonion#2592] [Unsupported Versions]
https://github.com/Security-Onion-Solutions/securityonion/discussions/2037

Based on this discussion I found when doing some research, it sounds like if I'm using a laptop with only one NIC, that NIC is probably being used by the management interface for SO, which would explain my issue:
**After completing setup on my Toshiba Satellite laptop, I can't find any networks on the WiFi menu to connect to -- which, to my understanding, means my SO machine is effectively monitoring nothing**

Should I seek to connect the laptop to my home router via a long ethernet cable; or would getting a wireless NIC device be adequate as well? The discussion I found seems to imply wired is the way to go; but I'm curious if I'm missing something or muddying the waters by basing too much of my issue on this other person's issue.

Thanks for reading!

**Comment by weslambert** [comment]
Your best bet is to configure a switch for port mirroring, and have SO connected to a SPAN port, or use a decent network TAP to obtain a copy of the network traffic.  You'll want it to occur before you hit the wireless access point, and you'll want SO to have more than one interface (for live monitoring; one for mgmt, one for monitoring).
